src/scripts/main.js

```diff
@@ -1,3 +1,30 @@
 'use strict';
 
-// write your code here
+// Get table and it's raws from HTML
+const table = document.querySelector('table');
+const rows = table.rows;
+
+// Create array from the copied data
+const columnToCopy = 1;
+const copiedData = [];
+
+for (let i = 0; i < rows.length; i++) {
+  const cell = rows[i].cells[columnToCopy];
+
+  copiedData.push(cell ? cell.textContent : '');
+}
+
+// Creating new row in HTML and insering data
+for (let i = 0; i < rows.length; i++) {
+  const newCell = rows[i].insertCell(rows[i].cells.length - 1);
```

Comment:
The method `insertCell()` should be used with the index where the new cell should be inserted. Using `rows[i].cells.length - 1` might not always insert the cell at the desired position, especially if the table has varying numbers of cells per row. Consider using `rows[i].cells.length` to append the cell at the end of each row.